app/assets/javascripts/components/story/ExpandedStory/index.js

```diff
@@ -55,12 +55,14 @@ export class ExpandedStory extends React.Component {
               story={story}
               titleRef={this.titleRef}
               onEdit={(newAttributes) => editStory(story.id, newAttributes)}
+              disabled={!Story.canEdit(story)}
```

Comment:
We should put this value in a variable and re-use it here and on line 65.

Reply:
Good idea.